gradle/dependencies.gradle

```diff
@@ -105,7 +107,10 @@ ext.dep = [
     kspGradlePlugin: "com.google.devtools.ksp:symbol-processing-gradle-plugin:1.5.20-1.0.0-beta04",
     kotlinxdatetime: "org.jetbrains.kotlinx:kotlinx-datetime:0.1.1",
     kotlinxserializationjson: "org.jetbrains.kotlinx:kotlinx-serialization-json:1.1.0",
-    atomicfu: "org.jetbrains.kotlinx:atomicfu:0.15.1"
+    atomicfu: "org.jetbrains.kotlinx:atomicfu:0.15.1",
+    ktor                  : [
+            clientJs: "io.ktor:ktor-client-js:$versions.ktor"
```

Comment:
Ultimately, I think we'll want to use `fetch` or some other API directly to save the dependency here. Can definitely be done in a later step though so 👍

Reply:
Have considered using Ktor for all platforms? Because then, you won't need to write all those platform specific HTTP/WS classes, because Ktor already does that for you.

Yeah, I've noticed the "API" you have is built mostly around OkHttp, si it won't be an easy task to change it everywhere.

Reply:
We could have a `KtorHttpEngine`, maybe in a separate artifact but I'd rather not add it as the default. 

Also, on Android/JVM, OkHttp is super widespread so users can keep their token interceptors, logging, etc...